packages/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt

```diff
@@ -88,6 +88,7 @@ class ExpoCameraView(
   private var pendingFaceDetectorSettings: Map<String, Any>? = null
   private var shouldDetectFaces = false
   private var mShouldScanBarCodes = false
+  public var defaultRecordOptions: Map<String, Any>? = null
```

Comment:
```suggestion
  private var defaultRecordOptions: Map<String?, Any?>? = null
```
It doesn't have to be public ;)

Also not that I've changed the type to be nullable.